pr-deploy/src/function/sa-pr-deploy-key.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This file should probably be `.gitignore`d (and a new private key generated now that it's been on the internets 🙃). For the other apps I added a "<config-file-name>.example.json" file that lists the fields and what should be written in them, and checked that file in instead.

The real config file is "our business"

Reply:
Yeah I got lazy here... GitGuardian told me the same thing. Where do you keep the actual key value?